src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -245,7 +248,30 @@ class AppsController(
       }
   }
 
-  private def deleteSingle(appId: PathId)(implicit identity: Identity): Route = ???
+  private def deleteSingle(appId: PathId)(implicit identity: Identity): Route =
+    forceParameter { force =>
+      lazy val notFound: Either[Rejection, RootGroup] =
```

Comment:
What is the motivation for this helper? Why not return `Left(Rejections.EntityNotFound.noApp(appId))` directly? ...ah, since it makes `deleteApp` more readable, how about moving it out of `deleteSingle`?